pkgs/top-level/go-packages.nix

```diff
@@ -796,6 +796,22 @@ let
     sha256 = "0yg1jpr7lcaqh6i8n9wbs9r128kk541qjv06r9a6fp9vj56rqr3m";
   };
 
+  gotty = buildFromGitHub {
+    rev     = "8f9d5ba582e4ad5e4ef41fe941cdd6280917aff1";
```

Comment:
Yes exactly, fetchFromGithub is the underlying fetcher and the archive API call is the same for revs/tags.